### PR TITLE
Portal 1322/subsmissions page filter options

### PR DESCRIPTION
**Pull request by tbolt**
# Description

* adds file submissions table
* adds structure for api calls (Zach Rizer)
* adds MSW mock service worker (Zach Rizer)
* adds mock data for fileSubmissions based off [Processing Status API ](https://github.com/CDCgov/data-exchange-processing-status/blob/develop/docs/processing-status-openapi-spec.yaml) (subject to change)

### CODE REVIEW

**Review by ian-sawyer — APPROVED** [review]
LGTM! Great job y'all :)

File: .env.example [hunk 1]
```diff
@@ -4,4 +4,6 @@ VITE_SAMS_AUTH_URL=
 VITE_SAMS_USER_INFO_URL=
 VITE_OAUTH_CALLBACK_URL=
 VITE_OAUTH_TOKEN_URL=
-VITE_UPLOAD_API_ENDPOINT=
\ No newline at end of file
+VITE_UPLOAD_API_ENDPOINT=
```
Comment (ZacharyRizer) [inline .env.example:7]: Perhaps we should rework this env var into the `VITE_API_BASE_URL` variable and the `config/api.ts` file? Keep all of the api routes under one env variable.

File: src/Submissions.tsx [hunk 2]
```diff
@@ -1,59 +1,149 @@
...
 function Submissions() {
+  const pageLimit = 10;
+  const [currentPageData, setCurrentPageData] = useState([]);
+  const [allData, setAllData] = useState([]);
+
+  useEffect(() => {
+    const fetchCall = async () => {
+      const res = await getFileSubmissions();
+
+      if (res.status != 200) return;
```
Comment (ZacharyRizer) [inline src/Submissions.tsx:26]: Not necessary for this ticket, but we should probably add some sort of visual feedback to the user if the api call fails. Some kind of modal, dialog box, spinner, etc.

File: src/config/api.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { getEnv } from "../utils/helperFunctions/env";
+
+const API_URL = getEnv("VITE_API_BASE_URL");
+
+const API_ENDPOINTS = {
+  fileSubmissions: `${API_URL}/fileSubmissions`,
```
Comment (ZacharyRizer) [inline src/config/api.ts:6]: This is where we would add the upload url (transferred from what was the env var).

**Review by ZacharyRizer — COMMENTED** [review]
Added a few small comments.

**Pull request merged**